new/LIT7078QeneEffo.xml

```diff
@@ -37,7 +38,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <abstract>
                 <p><foreign xml:lang="gez">Mawaddəs</foreign>-type <foreign xml:lang="gez">qǝne</foreign> 
                     attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
-                    in the manuscript.</p>
+                    in the manuscript. It appears also in the older manuscript <ref type="mss" corresp="BNFabb145">Paris, Bibliothèque nationale de France d'Abbadie 145</ref>,
```

Comment:
Maybe it's best clarify that the first manuscript refers to `<ref type="mss" corresp="UppQ1524o"/>`.